proxy/envoy/config.go

```diff
@@ -439,9 +443,17 @@ func buildOutboundListeners(mesh *proxyconfig.MeshConfig, sidecar proxy.Node, in
 	httpOutbound = buildEgressFromSidecarHTTPRoutes(mesh, instances, config, httpOutbound)
 
 	for port, routeConfig := range httpOutbound {
-		listeners = append(listeners,
-			buildHTTPListener(mesh, sidecar, instances, routeConfig, WildcardAddress, port,
-				fmt.Sprintf("%d", port), false, EgressTraceOperation))
+		var l *Listener
```

Comment:
nit: use variable to pass different trace operation, use remote address flag values (e.g. `useRemoteAddress := true; operation := EgressTraceOperation`)